### DISCUSSION TITLE
Reviewing translation after resource string update

### DISCUSSION THREAD

**Discussion by sshiells-scottlogic** [dotnet/roslyn#70652] [Q&A] [answered]
I am looking to start contributing and I was looking to tackle https://github.com/dotnet/roslyn/issues/67328 as a first issue.

After making my changes to CSharpCodeFixesResources.resx file there are a lot of changes to various translation files.  An example diff is given below: 

![image](https://github.com/dotnet/roslyn/assets/148051590/bfbae806-87b4-4fd1-9466-93ba8ba11243)

I was wondering what the process of getting these translations reviewed is? None of the actual translations have actually changed, just the casing in the source, and the target state.

**Comment by jasonmalinowski** [comment, marked as answer]
Just submit the change and it's OK to leave it as needs-review-translation; our translation teams pick up the changes and will submit an automatic update for the translation files.

**Reply by sshiells-scottlogic** [reply]
Perfect, I'll get that submitted tomorrow. Thanks!
